### PR TITLE
Fix #2816 : Add label for OptionsActivity

### LINKED ISSUE #2816: [A11y] Add label for OptionsActivity
## Task
Introduce label for `OptionsActivity` as `Options`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by starboi02**
## Explanation

Fixes #2816 : Add label for OptionsActivity

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+  }
+
+  private fun createOptionsActivityIntent(): Intent {
+    return OptionsActivity.createOptionsActivity(
+      ApplicationProvider.getApplicationContext(),
+      0,
+      false
+    )
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt]: If it can fit into a single line, then please do so.

**Review by prayutsu — CHANGES_REQUESTED** [review]
Thanks @starboi02, just suggest a nit change. PTAL.

**Comment by prayutsu** [comment]
Also, ensure that when your PR is ready for review, just leave a PTAL comment and mention the reviewers, so that they do not miss the PR to review.

File: app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  @Inject
+  lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testFaqListActivity_hasCorrectActivityLabel() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt]: The name of this test case is incorrect.

File: app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt (hunk 1 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt]: Suggest writing it with named arguments
```
    return OptionsActivity.createOptionsActivity(
      ApplicationProvider.getApplicationContext(),
      profileId = 0,
      isFromNavigationDrawer = false
    )
```

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL

File: app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt (hunk 2 above)
Comment (starboi02) [reply app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt]: @rt4914 done the change.

File: app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt (hunk 1 above)
Comment (starboi02) [reply app/src/sharedTest/java/org/oppia/android/app/options/OptionsActivityTest.kt]: @rt4914 done.

**Comment by starboi02** [comment]
@rt4914 @prayutsu PTAL. Made the changes suggested.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by prayutsu — APPROVED** [review]
LGTM!

**Review by BenHenning — APPROVED** [review]
Thanks @starboi02! I have nothing to add--this LGTM!

**Review by BenHenning — CHANGES_REQUESTED** [review]
Though, it appears there's a merge conflict. @starboi02 could you please merge with the latest changes from develop & address the merge conflict?

**Comment by starboi02** [comment]
@BenHenning I have resolved the merge conflict . PTAL.

**Review by BenHenning — APPROVED** [review]
Thanks @starboi02! LGTM.

**Pull request merged**